.github/bin/common.sh

```diff
@@ -406,3 +410,13 @@ waitWorkspaceStartedDevWorkspaceController() {
 createWorkspaceDevWorkspaceCheOperator() {
   oc apply -f https://raw.githubusercontent.com/che-incubator/devworkspace-che-operator/main/samples/flattened_theia-nodejs.yaml -n ${NAMESPACE}
 }
+
+deployEclipseCheStableWithChectl() {
```

Comment:
I would like to see something like `deployEclipseChe` with passing arguments to the function to make it reusable.